Cargo.toml

```diff
@@ -71,10 +71,10 @@ zk_os_api = { package = "zksync_os_api", git = "https://github.com/matter-labs/z
 #execution_utils = { package = "execution_utils", path = "../zksync-airbender/execution_utils" }
 #full_statement_verifier = { package = "full_statement_verifier", path = "../zksync-airbender/full_statement_verifier" }
 
-execution_utils = { package = "execution_utils", git = "https://github.com/matter-labs/zksync-airbender.git", tag = "v0.4.3" }
-full_statement_verifier = { package = "full_statement_verifier", git = "https://github.com/matter-labs/zksync-airbender.git", tag = "v0.4.3" }
+execution_utils = { package = "execution_utils", git = "https://github.com/matter-labs/zksync-airbender.git", tag = "v0.4.5" }
+full_statement_verifier = { package = "full_statement_verifier", git = "https://github.com/matter-labs/zksync-airbender.git", tag = "v0.4.5" }
 
-zksync_os_fri_prover = { git = "https://github.com/matter-labs/zksync-airbender-prover.git", tag = "v0.4.3" }
+zksync_os_prover_service = { git = "https://github.com/matter-labs/zksync-airbender-prover.git", branch = "ad-prover-service" }
```

Comment:
Let's merge this PR after the corresponding PR in prover is in `main` and a new tag is up